src/main/java/uk/gov/hmcts/divorce/systemupdate/event/SystemProgressCaseToAosOverdue.java

```diff
@@ -56,6 +56,8 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             applicationIssuedNotification.sendReminderToSoleRespondent(data, details.getId());
         }
 
+        applicationIssuedNotification.sendPartnerNotRespondedToSoleApplicant(data, details.getId());
```

Comment:
We should check whether this needs to be triggered if applicant1 is represented by a solicitor